Combining datasets (#51)
4TUResearchData-Carpentries/interoperability-climate-sciences: episodes/introduction.md

## Identify the three layers of interoperability

> These are properties that datasets must fulfilled in order to enable interoperability with the wider research ecosystem of APIs, notebooks, online viewers, etc. 

Reviewer: ```suggestion
These are properties that datasets must fulfill to enable interoperability with the wider research ecosystem, including APIs, notebooks, online viewers, and other tools.
```
